vnext/ReactUWP/packages.config

```diff
@@ -2,5 +2,4 @@
 <packages>
   <package id="boost" version="1.68.0.0" targetFramework="native" />
   <package id="boost_date_time-vc141" version="1.68.0.0" targetFramework="native" />
-  <package id="boost_system-vc141" version="1.68.0.0" targetFramework="native" />
```

Comment:
>oost_system-vc141 [](start = 16, length = 17)

Could you explain what changed which required this change ?

Reply:
This binary was not used at all.
It is built from sources included in the main boost package.